examples/labels.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any, Dict, List
+import pandas as pd
+
+
+class FeatureDefinition:
+    # Implement class logic
+    # TODO: Add a very specific type hint to the rule object:
+    # see: https://docs.python.org/3/library/typing.html#annotating-callable-objects
+    def __init__(self, label: str, rule: Dict[str, Any]):
+        if not callable(rule):
+            raise TypeError("Rule not callable")
+        # Immediately validate the rule, so you can error
+        # out/exit early if it's invalid
```

Comment:
In addition to checking the rule is callable, we also need to check whether the label is actually a string.

Take a look at the [`isinstance`](https://www.geeksforgeeks.org/python-isinstance-method/) method and see if you can come up with the correct check for label.